guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,31 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You download Ansible Galaxy from the AppStream RHEL 8 repository.
```

Comment:
The `ansible-core` package (which contains the `ansible-galaxy` binary) is automatically installed when you upgrade from EL7 to EL8 or when you deploy fresh on EL8, so this step is unnecessary.

Reply:
Any EL or RHEL specifically?

Reply:
Seriously, if they get galaxy, which contains the additional modules I presume, doesn't it render this whole module unnecessary? I'm confused.

Reply:
ansible-galaxy is the tool, Ansible Galaxy (aka galaxy.ansible.com) is the online service the tool talks to

Reply:
Okay, but I still don't understand what it implies for the text that is here and how I'm supposed to improve it. I'll talk to you on IRC.